examples/flutter_gallery/lib/demo/material/backdrop_demo.dart

```diff
@@ -0,0 +1,380 @@
+            onVerticalDragEnd: onVerticalDragEnd,
+            onTap: onTap,
+            child: new Container(
+              height: 48.0,
+              padding: const EdgeInsetsDirectional.only(start: 16.0),
+              alignment: AlignmentDirectional.centerStart,
+              child: new DefaultTextStyle(
+                style: theme.textTheme.subhead,
+                child: title,
+              ),
+            ),
+          ),
+          const Divider(height: 1.0),
+          new Expanded(child: child),
+        ],
+      ),
+    );
+  }
+}
+
+class BackdropTitle extends AnimatedWidget {
+  const BackdropTitle({
+    Key key,
+    Listenable listenable,
+  }) : super(key: key, listenable: listenable);
+
+  @override
+  Widget build(BuildContext context) {
+    final Animation<double> animation = listenable;
+    return new DefaultTextStyle(
+      style: Theme.of(context).primaryTextTheme.title,
+      softWrap: false,
+      overflow: TextOverflow.ellipsis,
+      child: new Stack(
+        children: <Widget>[
+          new Opacity(
+            opacity: new CurvedAnimation(
+              parent: new ReverseAnimation(animation),
+              curve: const Interval(0.5, 1.0),
+            ).value,
+            child: const Text('Select a Category'),
+          ),
+          new Opacity(
+            opacity: new CurvedAnimation(
+              parent: animation,
+              curve: const Interval(0.5, 1.0),
+            ).value,
+            child: const Text('Asset Viewer'),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class BackdropDemo extends StatefulWidget {
+  static const String routeName = '/material/backdrop';
+
+  @override
+  _BackdropDemoState createState() => new _BackdropDemoState();
+}
+
+class _BackdropDemoState extends State<BackdropDemo> with SingleTickerProviderStateMixin {
+  final GlobalKey _backdropKey = new GlobalKey(debugLabel: 'Backdrop');
+  AnimationController _controller;
+  Category _category = allCategories[0];
+
+  @override
+  void initState() {
+    super.initState();
+    _controller = new AnimationController(
+      duration: const Duration(milliseconds: 300),
+      value: 1.0,
+      vsync: this,
+    );
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  void _changeCategory(Category category) {
+    setState(() {
+      _category = category;
+      _controller.fling(velocity: 2.0);
+    });
+  }
+
+  bool get _backdropPanelVisible {
+    final AnimationStatus status = _controller.status;
+    return status == AnimationStatus.completed || status == AnimationStatus.forward;
+  }
+
+  void _toggleBackdropPanelVisibility() {
+    _controller.fling(velocity: _backdropPanelVisible ? -2.0 : 2.0);
+  }
+
+  double get _backdropHeight {
+    final RenderBox renderBox = _backdropKey.currentContext.findRenderObject();
+    return renderBox.size.height;
+  }
+
+  // By design: the panel can only be opened with a swipe. To close the panel
```

Comment:
It can't be closed by swipe?

Reply:
Our UX designer was really clear about that, yes.